packages/app/src/settings/project/renderers/RenderObject.vue

```diff
@@ -46,7 +46,10 @@
     <span
       :class="props.colorClasses"
       :data-cy-config="props.from"
-    >{</span><br><template
+    >{</span>
+    <!-- Do not render break line for empty object or array -->
+    <br v-if="Array.isArray(props.value) ? props.value.length : Object.keys(props.value).length">
```

Comment:
will `Array.isArray(props.value)` ever be hit? It looks like the first template is used above when `<template v-if="Array.isArray(value)">`

Reply:
It should not but https://github.com/cypress-io/cypress/pull/25053#discussion_r1043746922 misleaded me.

Reply:
ohh @mike-plummer do you think this is needed?

Reply:
@emilyrohrbough Yeah, should be safe to remove. I didn't originally see that we had a separate template section for when `value` is an array

Reply:
@emilyrohrbough done